roles/openshift_autoheal/tasks/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+- include_tasks: install.yml
+  when: openshift_autoheal_deploy | bool
+
+- include_tasks: uninstall.yml
+  when: not openshift_autoheal_deploy | bool
```

Comment:
I agree. if `openshift_autoheal_deploy` is false by default this will cause the openshift_autoheal uninstall task file to run on every install of openshift-ansible. It wouldn't do any damage, but it will slightly slow down the install by doing unnecessary stuff.

Reply:
The playbook never gets called unless openshift_autoheal_deploy is set to True during the main install of the cluster.

This would only apply if the install/upgrade play was called directly, in which case, defaulting to false would be fine IMO.